### DISCUSSION TITLE
Mastering Pytorch

### DISCUSSION THREAD

**Discussion by PhdPIML** [MarsArtistsCommunity/CommunityOverview#9] [Q&A] [answered]
Which would your recommend a person cram this weekend? Rust for Rustaceans or Mastering PyTorch?

**Comment by deliquentspacexer** [comment]
WASM and Dioxus would be useful for our team

**Comment by PhdPIML** [comment]
Yes I'm looking into Dioxus and Next.js too, but, anything explicitly for PyTorch?

**Comment by oceanbluesky** [comment, marked as answer]
https://pytorch.org/tutorials/

https://docs.modular.com/mojo/manual/get-started

https://rustwasm.github.io/book/game-of-life/introduction.html

Mastering PyTorch...go slow, we're going to be around for a while just make yourself useful to be invited back, build a foundation
